shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/utility/Extensions.kt

```diff
@@ -0,0 +1,10 @@
+package com.kcc.kmmhackathon.shared.utility
+import kotlin.math.round
+
+fun Double.round(decimals: Int): Double {
+    var multiplier = 1.0
+    repeat(decimals) { multiplier *= 10 }
+    return round(this * multiplier) / multiplier
+}
+
+fun Double.toKilometres(): Double = this * 1.609344
```

Comment:
I don't know what the best practices are with this but in iOS I would be wary of having a file with functions floating around that don't belong to anything in particular